core/base/src/TAttText.cxx

```diff
@@ -229,21 +229,21 @@ printout given by this command:
 ### List of the currently supported fonts
 
 ~~~ {.cpp}
-   Font number         X11 Names             Win32/TTF Names
-       1 :       times-medium-i-normal      "Times New Roman"
-       2 :       times-bold-r-normal        "Times New Roman"
-       3 :       times-bold-i-normal        "Times New Roman"
-       4 :       helvetica-medium-r-normal  "Arial"
-       5 :       helvetica-medium-o-normal  "Arial"
-       6 :       helvetica-bold-r-normal    "Arial"
-       7 :       helvetica-bold-o-normal    "Arial"
-       8 :       courier-medium-r-normal    "Courier New"
-       9 :       courier-medium-o-normal    "Courier New"
-      10 :       courier-bold-r-normal      "Courier New"
-      11 :       courier-bold-o-normal      "Courier New"
-      12 :       symbol-medium-r-normal     "Symbol"
-      13 :       times-medium-r-normal      "Times New Roman"
-      14 :                                  "Wingdings"
+   Font number         X11 Names             TTF Names                  PDF Names
```

Comment:
The new names look good, but could we add some documentation or links to the original source for `X11` names or `TTF` or `PDF` names. From my perspective, reading `TTF Names` or the others doesn't really tell me much about these names. Who decided what `TTF Names` are? Is there some documentation somewhere in the original protocols/repositories? Similarly for X11 and PDF